nep-Y.mediawiki

```diff
@@ -210,11 +210,11 @@ Neo-specific codes are documented here, generic JSON-RPC errors like -32602 or
 |-
 | getapplicationlog   || -105
 |-
-| getblock            || -101
+| getblock            || -101, -109
 |-
-| getblockhash        || -109
+| getblockhash        || -101, -109
```

Comment:
Why -101 here? It just resolves height to hash, so invalid params or wrong height are expected, but -101 is not.

Reply:
Removed.